storage/test/transfer.test.js

```diff
@@ -0,0 +1,639 @@
+
+        sample.mocks.storagetransfer.transferJobs.list.callsArgWith(1, null, {});
+        sample.program.listTransferJobs(callback);
+
+        assert(callback.calledTwice, 'callback called twice');
+        assert.equal(callback.secondCall.args.length, 2, 'callback received 2 arguments');
+        assert.ifError(callback.secondCall.args[0], 'callback did not receive error');
+        assert.deepEqual(callback.secondCall.args[1], [], 'callback received no transfer jobs');
+      });
+
+      it('should handle auth error', function () {
+        var error = new Error('error');
+        var sample = getSample();
+        var callback = sinon.stub();
+        sample.mocks.googleapis.auth.getApplicationDefault.callsArgWith(0, error);
+
+        sample.program.listTransferJobs(callback);
+
+        assert(callback.calledOnce, 'callback called once');
+        assert.equal(callback.firstCall.args.length, 1, 'callback received 1 argument');
+        assert(callback.firstCall.args[0], 'callback received error');
+        assert.equal(callback.firstCall.args[0].message, error.message, 'error has correct message');
+      });
+
+      it('should handle list error', function () {
+        var error = new Error('error');
+        var sample = getSample();
+        var callback = sinon.stub();
+        sample.mocks.storagetransfer.transferJobs.list.callsArgWith(1, error);
+
+        sample.program.listTransferJobs(callback);
+
+        assert(callback.calledOnce, 'callback called once');
+        assert.equal(callback.firstCall.args.length, 1, 'callback received 1 argument');
+        assert(callback.firstCall.args[0], 'callback received error');
+        assert.equal(callback.firstCall.args[0].message, error.message, 'error has correct message');
+      });
+    });
+  });
+
+  describe('operations', function () {
+    describe('list', function () {
+      it('should list transfer operations', function () {
+        var sample = getSample();
+        var callback = sinon.stub();
+
+        sample.program.listTransferOperations(undefined, callback);
+
+        assert(callback.calledOnce, 'callback called once');
+        assert.equal(callback.firstCall.args.length, 2, 'callback received 2 arguments');
+        assert.ifError(callback.firstCall.args[0], 'callback did not receive error');
+        assert.deepEqual(callback.firstCall.args[1], [sample.mocks.transferOperation], 'callback received transfer operations');
+
+        sample.program.listTransferOperations(jobName, callback);
+
+        assert(callback.calledTwice, 'callback called twice');
+        assert.equal(callback.secondCall.args.length, 2, 'callback received 2 arguments');
+        assert.ifError(callback.secondCall.args[0], 'callback did not receive error');
+        assert.deepEqual(callback.secondCall.args[1], [sample.mocks.transferOperation], 'callback received transfer operations');
```

Comment:
These seem redundant - the second one includes a `jobName` parameter, but doesn't seem to test anything new.